ext/fileinfo/fileinfo.c

```diff
@@ -276,7 +268,7 @@ PHP_FUNCTION(finfo_set_flags)
 	}
 	FILEINFO_FROM_OBJECT(finfo, self);
 
-	FINFO_SET_OPTION(finfo->magic, options)
+	magic_setflags(finfo->magic, options);
```

Comment:
Can you add a short comment here and below explaining why you don't check the return value.
Not necessary for the restore code.